llvm/runtimes/CMakeLists.txt

```
@@ -17,7 +17,11 @@ foreach(proj ${LLVM_ENABLE_RUNTIMES})
 endforeach()
 
 function(get_compiler_rt_path path)
-  foreach(entry ${runtimes})
+  set(all_runtimes ${runtimes})
+  foreach(name ${LLVM_RUNTIME_TARGETS})
+    list(APPEND all_runtimes RUNTIMES_${name}_LLVM_ENABLE_RUNTIMES)
+  endforeach()
```

You should consider deduplicating the items in the list here:
```suggestion
  endforeach()
  list(REMOVE_DUPLICATES all_runtimes)
```